app/src/main/java/com/rentminder/dto/Payment.kt

```diff
@@ -1,6 +1,6 @@
 package com.rentminder.dto
 
-data class Payment (var paymentId: Int = 0, var memberId: Int, var paymentType: String, var paymentNum: Double){
+data class Payment (val paymentId: Int = 0, val memberId: Int, val paymentType: String, val paymentNum: Double){
```

Comment:
(Repeated) Make sure to use vals instead of vars in data classes as it is good practice to use immutable properties of instances. URL: https://www.digitalocean.com/community/tutorials/kotlin-data-class